assets/data_table/main.js

```diff
@@ -101,9 +101,11 @@ function App({ ctx, data }) {
   });
 
   const hasRefetch = data.features.includes("refetch");
+  const hasExport = data.features.includes("export");
   const hasData = data.content.columns.length !== 0;
   const hasSummaries = summariesItems.length > 0;
   const hasSorting = data.features.includes("sorting");
+  const supportedFormats = data.info.export || ["csv"];
```

Comment:
I think we should make the export info required if `:export` feature is enabled, this way it's clear they must support the formats they specify.